CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## UNRELEASED
+
+* Bugfix: Ensure cookie banner hide button always hides the cookie banner.
```

Comment:
```suggestion
* Ensure cookie banner hide button always hides the cookie banner. ([PR #1443](https://github.com/alphagov/govuk_publishing_components/pull/1443))
```

Reply:
Needs a rebase before merging for the changelog (as another PR got merged meanwhile)